src/disk.c

```diff
@@ -164,8 +167,9 @@ static char *conf_udev_name_attr;
 static struct udev *handle_udev;
 #endif
 
-static const char *config_keys[] = {"Disk", "UseBSDName", "IgnoreSelected",
-                                    "UdevNameAttr"};
+static const char *config_keys[] = {"Disk",           "UseBSDName",
+                                    "IgnoreSelected", "UdevNameAttr",
+                                    "ReportDiscard",  "ReportFlush"};
```

Comment:
Please add the new config options to the documentation:

* `src/collectd.conf.pod` (the *collectd.conf(5)* manpage)
* `src/collectd.conf.in` (the default config; add options with their default value there)